src/Microsoft.DotNet.Build.Tasks/GenerateEncodingTable.cs

```diff
@@ -0,0 +1,395 @@
+﻿using Microsoft.Build.Framework;
```

Comment:
Yes I think we should avoid this work as part of everyone's build if we can and only run the update on demand.